zebra-chain/src/orchard/tree.rs

```diff
@@ -228,6 +261,49 @@ impl NoteCommitmentTree {
     pub fn uncommitted() -> pallas::Base {
         pallas::Base::one().double()
     }
+
+    /// Count of note commitments added to the tree.
+    ///
+    /// For Orchard, the tree is capped at 2^32.
+    pub fn count(&self) -> u32 {
```

Comment:
This code was copied from librustzcash - we want to avoid modifying it as much as possible.

```suggestion
    /// For Orchard, the tree is capped at 2^32.
    //
    // This code is based on https://github.com/zcash/librustzcash/blob/4c4d226f404aef64df6a77caa289b30cc875ae4f/zcash_primitives/src/merkle_tree.rs#L91
    pub fn count(&self) -> u32 {
```